[2025-01-24T19:13:06.799+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T19:13:06.813+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-23T00:00:00+00:00 [queued]>
[2025-01-24T19:13:06.820+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-23T00:00:00+00:00 [queued]>
[2025-01-24T19:13:06.821+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-01-24T19:13:06.833+0000] {taskinstance.py:2888} INFO - Executing <Task(InitFoldersTask): init_folders_task> on 2025-01-23 00:00:00+00:00
[2025-01-24T19:13:06.848+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=102) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-24T19:13:06.850+0000] {standard_task_runner.py:72} INFO - Started process 147 to run task
[2025-01-24T19:13:06.850+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pull_gazettes_data', 'init_folders_task', 'scheduled__2025-01-23T00:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmpj98vue74']
[2025-01-24T19:13:06.852+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask init_folders_task
[2025-01-24T19:13:06.889+0000] {task_command.py:467} INFO - Running <TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-23T00:00:00+00:00 [running]> on host 5792905bcb6a
[2025-01-24T19:13:06.963+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pull_gazettes_data' AIRFLOW_CTX_TASK_ID='init_folders_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-23T00:00:00+00:00'
[2025-01-24T19:13:06.964+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T19:13:06.991+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T19:13:06.992+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=pull_gazettes_data, task_id=init_folders_task, run_id=scheduled__2025-01-23T00:00:00+00:00, execution_date=20250123T000000, start_date=20250124T191306, end_date=20250124T191306
[2025-01-24T19:13:07.025+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T19:13:07.048+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-24T19:13:07.050+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-01-24T20:00:16.943+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T20:00:16.968+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-23T00:00:00+00:00 [queued]>
[2025-01-24T20:00:16.980+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-23T00:00:00+00:00 [queued]>
[2025-01-24T20:00:16.981+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-01-24T20:00:16.998+0000] {taskinstance.py:2888} INFO - Executing <Task(InitFoldersTask): init_folders_task> on 2025-01-23 00:00:00+00:00
[2025-01-24T20:00:17.024+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=422) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-24T20:00:17.029+0000] {standard_task_runner.py:72} INFO - Started process 495 to run task
[2025-01-24T20:00:17.027+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pull_gazettes_data', 'init_folders_task', 'scheduled__2025-01-23T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmpyntzhtsy']
[2025-01-24T20:00:17.031+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask init_folders_task
[2025-01-24T20:00:17.107+0000] {task_command.py:467} INFO - Running <TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-23T00:00:00+00:00 [running]> on host 972b45cbfb12
[2025-01-24T20:00:17.224+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pull_gazettes_data' AIRFLOW_CTX_TASK_ID='init_folders_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-23T00:00:00+00:00'
[2025-01-24T20:00:17.226+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T20:00:17.279+0000] {logging_mixin.py:190} INFO - Folder: /usr/local/***/data/pdf_single_page/ does not exists, creating it.
[2025-01-24T20:00:17.298+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T20:00:17.299+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=pull_gazettes_data, task_id=init_folders_task, run_id=scheduled__2025-01-23T00:00:00+00:00, execution_date=20250123T000000, start_date=20250124T200016, end_date=20250124T200017
[2025-01-24T20:00:17.366+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T20:00:17.399+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-24T20:00:17.401+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-01-24T21:11:00.025+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T21:11:00.045+0000] {taskinstance.py:2602} INFO - Dependencies not met for <TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-23T00:00:00+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2025-01-24T21:11:00.047+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
[2025-01-24T23:13:11.740+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T23:13:11.764+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-23T00:00:00+00:00 [queued]>
[2025-01-24T23:13:11.780+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-23T00:00:00+00:00 [queued]>
[2025-01-24T23:13:11.780+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-01-24T23:13:11.800+0000] {taskinstance.py:2888} INFO - Executing <Task(InitFoldersTask): init_folders_task> on 2025-01-23 00:00:00+00:00
[2025-01-24T23:13:11.827+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=371) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-24T23:13:11.830+0000] {standard_task_runner.py:72} INFO - Started process 430 to run task
[2025-01-24T23:13:11.832+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pull_gazettes_data', 'init_folders_task', 'scheduled__2025-01-23T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmpevdrqpyx']
[2025-01-24T23:13:11.837+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask init_folders_task
[2025-01-24T23:13:11.920+0000] {task_command.py:467} INFO - Running <TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-23T00:00:00+00:00 [running]> on host 41bcec3d818b
[2025-01-24T23:13:12.069+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pull_gazettes_data' AIRFLOW_CTX_TASK_ID='init_folders_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-23T00:00:00+00:00'
[2025-01-24T23:13:12.072+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T23:13:12.129+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T23:13:12.130+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=pull_gazettes_data, task_id=init_folders_task, run_id=scheduled__2025-01-23T00:00:00+00:00, execution_date=20250123T000000, start_date=20250124T231311, end_date=20250124T231312
[2025-01-24T23:13:12.208+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T23:13:12.246+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-24T23:13:12.249+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
